kits/flutter_ume_kit_ui/lib/components/color_picker/color_picker.dart

```diff
@@ -75,16 +81,18 @@ class _ColorSuckerState extends State<ColorSucker> {
   }
 
   void _onPanUpdate(DragUpdateDetails dragDetails) {
-    _magnifierPosition =
-        dragDetails.globalPosition - _magnifierSize.center(Offset.zero);
-    double newX = dragDetails.globalPosition.dx;
-    double newY = dragDetails.globalPosition.dy;
+    _magnifierPosition = dragDetails.globalPosition -
+        _magnifierSize.center(Offset(_handlerSize.width, _handlerSize.height));
+    _handlerPosition =
+        dragDetails.globalPosition - _handlerSize.center(Offset.zero);
+    double newX = dragDetails.globalPosition.dx - _magnifierSize.width / 2;
+    double newY = dragDetails.globalPosition.dy - _magnifierSize.height / 2;
     final Matrix4 newMatrix = Matrix4.identity()
       ..translate(newX, newY)
       ..scale(_scale, _scale)
       ..translate(-newX, -newY);
     _matrix = newMatrix;
-    _searchPixel(dragDetails.globalPosition);
+    _searchPixel(Offset(newX, newY));
```

Comment:
I tested it on multiple devices and the color picking point does not work correctly. Just like the screenshot.
![IMG_AAB2C280482E-1](https://user-images.githubusercontent.com/11439167/127826375-0a533975-ed4d-43c8-8799-663c4f1d5a0d.jpeg)

Reply:
I think it should be known issue in original code, not cause by the PR. @smileShirely

Reply:
![image](https://user-images.githubusercontent.com/11439167/128500555-553b7af9-cbc2-49c9-82b7-7d90be872a84.png)
Sorry.. I can't agree. The screenshot shows the same pixel has difference RGB values between yours and the original one.

Reply:
Sorry for the delay, you can see the ss below, I am using the original code and test it in one plus 8T. 

<img src="https://raw.githubusercontent.com/KevinZhang19870314/flutter_ume/master_color_sucker_issue/screenshots/color_sucker_issue.jpg" />